releases/v20.1.0-alpha.20200210.md

```diff
@@ -0,0 +1,351 @@
+- An optimization has been added to scan over only 1 row when finding the MIN/MAX of a single aggregate group, as long as the correct index is present. [#43547][#43547]
+- [`SHOW CREATE TABLE`](../v20.1/show-create.html) now also emits the `COMMENT` statements sufficient to populate the table's user-defined comments, if any, alongside the `CREATE` statement proper. [#43152][#43152]
+- More invalid usages of `FOR UPDATE` [locking clauses](../v20.1/sql-grammar.html#locking_clause) are now rejected by the SQL optimizer. [#44015][#44015]
+- Added the `timeofday` functionality supported by PostgreSQL, which returns the time on one of the nodes as a formatted string. [#44050][#44050]
+- Added `localtime`, which by default returns the current time as the [`TIME`](../v20.1/time.html) data type (as opposed to `current_time`, which returns the [`TIMETZ`](../v20.1/time.html) data type). [#44042][#44042]
+- Added `localtimestamp`, which by default returns the current timestamp as the [`TIMESTAMP`](../v20.1/timestamp.html) data type (as opposed to `current_timestamp`, which returns the [`TIMESTAMPTZ`](../v20.1/timestamp.html) data type). [#44042][#44042]
+- Added support for having `D:H:M`, `D:M:S.fff`, or `D:H:M:S.fff` for interval parsing if the first element is a decimal or empty (e.g., `:04:05` and `1.0:04:05` would be `04:05:00` and `1 day 04:05:00` respectively). [#43924][#43924]
+- Previously, floats were supported in `H:M:S` formats (e.g., `1.0:2.0:3.0`), which did not make sense. Floats are no longer allowed for the M field. [#43924][#43924]
+- Previously, CockroachDB would return an internal error when using `SET tracing` with any type other than string. Now it will return a regular query error. Additionally, boolean arguments are now supported in `SET tracing`, and `true` is mapped to `on` mode of tracing whereas `false` is mapped to `off`. [#44260][#44260]
+- Indexes that reference, or are referenced by, a foreign key constraint can now be dropped if there is another suitable index for the constraint. [#43332][#43332]
+- Added a `log` builtin for any base (e.g., `log(2.0, 4.0)`). [#41848][#41848]
+- Non-admin users can now query [`SHOW JOBS`](../v20.1/show-jobs.html) and `crdb_internal.jobs` and see their own jobs. [#44345][#44345]
+- Vectorized execution engine now supports `DISTINCT` on unordered input. [#42522][#42522]
+- Primary key changes can now be performed on interleaved children. [#44075][#44075]
+- Primary key changes are now enabled on tables that have foreign key relationships. [#43830][#43830]
+- `pg_catalog` access method information is now more accurate. Added inverted index to the access methods listed in `pg_am` and set `pg_class.relam` to zero for sequences and views, which is more consistent with PostgreSQL. [#43715][#43715]
+- An overload has been added to the `unnest` builtin function in order to support multiple array arguments. [#41557][#41557]
```

Comment:
This and other mentions of built-in functions in this doc could link to https://www.cockroachlabs.com/docs/dev/functions-and-operators.html